src/i18n/en.ts

```diff
@@ -225,9 +225,12 @@ const en = {
         "resource.metadata.file.content.view.tooltip": "View file content and annotate it",
         "resource.metadata.file.content.download": "Download",
         "resource.metadata.document.vocabulary": "Document vocabulary",
-        "resource.metadata.document.files.add": "Add file",
-        "resource.metadata.document.files.create.tooltip": "Add new file to this document",
-        "resource.metadata.document.files.create.dialog.title": "New file",
+        "resource.metadata.document.files.actions.add": "Add file",
+        "resource.metadata.document.files.actions.add.tooltip": "Add new file to this document",
+        "resource.metadata.document.files.actions.add.dialog.title": "New file",
+        "resource.metadata.document.files.actions.remove": "Remove file",
+        "resource.metadata.document.files.actions.remove.tooltip": "Remove file",
+        "resource.metadata.document.files.actions.remove.dialog.title": "Do You Really want to remove the file?",
```

Comment:
Letter casing in this sentence makes no sense to me.

Reply:
Also, same as in the Czech version - why not reuse the existing message "asset.remove.dialog.title"?